cfme/containers/provider/openshift.py

```diff
@@ -113,6 +113,12 @@ def __init__(
 
     def create(self, **kwargs):
 
+        # Enable C&U for any provider with metrics provider
+        if getattr(self, "metrics_type") != "Disabled":
```

Comment:
**REQUIRED**
if a provider doesn't have "metrics_type" attribute  `AttributeError` will be raised.

Reply:
I had this discussion with @mshriver last week, we agreed that all the OpenShift Providers MUST have this attribute and if it missing then we have to fix the YAML accordingly, therefore I didn't set a default value to ```getattr```

Reply:
@quarckster there's a default of None for metrics_type attribute, so will always be there.

Reply:
@shalomnaim1 @mshriver  In that case `getattr` doesn't make sense, use `self.metrics_type` instead.

Reply:
@quarckster Ok, that's make seanse